packages/mui/README.md

```diff
@@ -58,12 +58,12 @@ See the [Iconography](https://brightlayer-ui.github.io/style/iconography) on bri
 To build the icons-mui package (e.g., after new icons are added to Brightlayer UI):
 
 ```
-git clone https://github.com/brightlayer-ui/icons-mui
-cd icons-mui
-yarn build:full
+git clone hhttps://github.com/brightlayer-ui/icons/tree/master/packages/mui
+cd packages/mui
+yarn build
 ```
 
-This will populate the /icons directory with all of the icon components as well as copy over necessary files for publishing (Readme, package.json, etc.).
+This will populate the /icon-font directory with all of the icon components as well as copy over necessary files for publishing (Readme, package.json, etc.).
```

Comment:
```suggestion
This will populate the /dist folder with all of the icon components as well as copy over necessary files for publishing (Readme, package.json, etc.).
```